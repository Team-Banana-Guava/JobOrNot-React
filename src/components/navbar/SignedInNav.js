import React, { PropTypes } from 'react';

const SignedInNav = (props) => {
  return (
    <nav>
      <ul>
        <li><img src="http://placehold.it/50x50" alt='logo' /></li>
        <li>JobOrNot</li>
        <li>How It Works</li>
        <li>Profile</li>
        <li>{props.button}</li>
      </ul>
    </nav>
  )
}

SignedInNav.propTypes = {
  button: PropTypes.node.isRequired,
}

export default SignedInNav;