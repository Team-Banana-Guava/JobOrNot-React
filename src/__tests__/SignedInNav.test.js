/* eslint-disable */
import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import SignedInNav from '../components/navbar/SignedInNav';

describe('SignedInNav', function () {
    it('renders the SignedInNav component', () => {
        const rendered = shallow(<SignedInNav button={<button>button node</button>} />);
        expect(toJson(rendered)).toMatchSnapshot();
    });
});
