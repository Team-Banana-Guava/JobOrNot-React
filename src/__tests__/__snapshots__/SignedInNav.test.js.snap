exports[`SignedInNav renders the SignedInNav component 1`] = `
<nav>
  <ul>
    <li>
      <img
        alt="logo"
        src="http://placehold.it/50x50" />
    </li>
    <li>
      JobOrNot
    </li>
    <li>
      How It Works
    </li>
    <li>
      Profile
    </li>
    <li>
      <button>
        button node
      </button>
    </li>
  </ul>
</nav>
`;
